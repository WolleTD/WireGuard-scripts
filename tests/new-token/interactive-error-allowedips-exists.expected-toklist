[A-Z2-7]{8}		172\.16\.1\.2/32		peer2
