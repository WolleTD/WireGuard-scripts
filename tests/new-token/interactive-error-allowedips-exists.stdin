peer2
172.16.1.1/32
peer2
172.16.1.2/32
n
