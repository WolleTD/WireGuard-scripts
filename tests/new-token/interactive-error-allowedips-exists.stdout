Add wg-setup tokens to /tmp/tmp\..+/toklist
Name: AllowedIPs: A peer with this AllowedIPs already exists!
Name: AllowedIPs: 
[A-Z2-7]{8}		172\.16\.1\.2/32		peer2
Add another token\? \[y/N\] 
